frontend/src/components/authchooser/index.tsx

```diff
@@ -80,15 +91,16 @@ function AuthChooser({ children }: AuthChooserProps) {
         setTestingAuth(true);
 
         let errorObj: Error | null = null;
-        setError(errorObj);
+
+        console.debug('Testing auth at authchooser');
```

Comment:
console.debug left in on purpose?

Reply:
yes. If we see millions of prints, we'll know :)